Tests/Benchmarks/BoundaryToleranceBenchmark.cpp

```diff
@@ -26,7 +22,8 @@ int main(int /*argc*/, char** /*argv[]*/) {
   // === PROBLEM DATA ===
 
   // Trapezoidal area of interest
-  const Vector2 poly[] = {{0.4, 0.25}, {0.6, 0.25}, {0.8, 0.75}, {0.2, 0.75}};
+  ConvexPolygonBounds<PolygonDynamic> poly(
```

Comment:
Isn't this switching from compile time size to runtime size here? Probably fine but I guess it changes the benchmarks.

Reply:
the code below uses the polygon in a dynamic context so the benchmark should be unaffected